internal/cmd/push.go

```diff
@@ -0,0 +1,193 @@
+- GET /api/v1/manifest - Fetches existing flags for comparison
+
+Remote services implementing this API should accept the flag data in the format
+specified by the OpenFeature flag manifest schema.
+
+Note: The file:// scheme is not supported for push operations.
+For local file operations, use standard shell commands like cp or mv.`,
+		Example: `  # Push flags to a remote HTTPS endpoint (smart push: creates and updates as needed)
+  openfeature push --flag-source-url https://api.example.com --auth-token secret-token
+
+  # Push flags to an HTTP endpoint (development)
+  openfeature push --flag-source-url http://localhost:8080
+
+  # Dry run to preview what would be sent
+  openfeature push --flag-source-url https://api.example.com --dry-run`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return initializeConfig(cmd, "push")
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Get configuration values
+			flagSourceUrl := config.GetFlagSourceUrl(cmd)
+			manifestPath := config.GetManifestPath(cmd)
+			authToken := config.GetAuthToken(cmd)
+			dryRun := config.GetDryRun(cmd)
+
+			// Validate destination URL is provided
+			if flagSourceUrl == "" {
+				return fmt.Errorf("flag source URL is required. Please provide --flag-source-url")
+			}
+
+			// Parse and validate URL
+			parsedURL, err := url.Parse(flagSourceUrl)
+			if err != nil {
+				return fmt.Errorf("invalid source URL: %w", err)
+			}
+
+			// Load the local manifest
+			flags, err := manifest.LoadFlagSet(manifestPath)
+			if err != nil {
+				return fmt.Errorf("error loading manifest from %s: %w", manifestPath, err)
+			}
+
+			// Validation of required fields is handled by manifest.LoadFlagSet
+
+			// Handle URL schemes
+			switch parsedURL.Scheme {
+			case "file":
+				return fmt.Errorf("file:// scheme is not supported for push. Use standard shell commands (cp, mv) for local file operations")
+			case "http", "https":
+				// Perform smart push (fetches remote, compares, and creates/updates as needed)
+				// In dry run mode, performs comparison but skips actual API calls
+				result, err := manifest.SaveToRemote(flagSourceUrl, flags, authToken, dryRun)
+				if err != nil {
+					return fmt.Errorf("error pushing flags to remote destination: %w", err)
+				}
+
+				// Display the results
+				displayPushResults(result, flagSourceUrl, dryRun)
+			default:
+				return fmt.Errorf("unsupported URL scheme: %s. Supported schemes are http:// and https://", parsedURL.Scheme)
+			}
+
+			return nil
+		},
+	}
+
+	// Add push-specific flags
+	config.AddPushFlags(pushCmd)
+
+	// Add common flags (like --manifest)
+	config.AddRootFlags(pushCmd)
+
+	return pushCmd
+}
+
+// displayPushResults renders the push operation results with color-coded output
+// If dryRun is true, displays what would be pushed instead of what was pushed
+func displayPushResults(result *sync.PushResult, destination string, dryRun bool) {
```

Comment:
It would be nice if the destination were only the domain. For example, if I configure the `flag-sync-url` to be `https://flagd-studio.onrender.com/api/`, I'd like the output of this message to be `https://flagd-studio.onrender.com/`. That would allow me to click the link in the console to quickly open the remote flag management system.

You should be able to use the [net/url package](https://pkg.go.dev/net/url) to do this.